game/view/gameplay/actionhud/init.lua

```diff
@@ -109,15 +105,29 @@ function ActionHUD:init(route)
   ADJACENCY.unset(self.adjacency)
 
   -- Control hints
-  self.cancel_hint = CancelHint(13, 8)
+  self.hand_hint = ControlHint(240+13, 28, "button-key-draw_hand", {
+    DEFAULT = "Open hand", FOCUS = "Close hand"
+  })
+  self.hand_hint:register("HUD_MIDDLE")
```

Comment:
HUD_MIDDLE could be a value on a table somewhere

Reply:
Added issue #1463 for this specific type of magical literal.